x-pack/plugins/apm/server/routes/alerts/register_anomaly_alert_type.ts

```diff
@@ -130,6 +134,14 @@ export function registerAnomalyAlertType({
           return {};
         }
 
+        // Window size must be at least 30, does like this to support rules created before this change where default was 15
+        const windowSize = Math.min(
+          datemath.parse('now-30m')!.valueOf(),
+          datemath
+            .parse(`now-${ruleParams.windowSize}${ruleParams.windowUnit}`)
+            ?.valueOf() || 0
```

Comment:
`windowSize` is probably the wrong term here, it's the start of the time range we want to query.